diff -up ./libraries/classes/Utils/HttpRequest.php.rpm ./libraries/classes/Utils/HttpRequest.php
--- ./libraries/classes/Utils/HttpRequest.php.rpm	2019-09-12 15:57:49.955739598 +0200
+++ ./libraries/classes/Utils/HttpRequest.php	2019-09-12 15:58:00.262698588 +0200
@@ -141,12 +141,12 @@ class HttpRequest
          *
          * See https://letsencrypt.org/certificates/
          */
-        $certsDir = ROOT_PATH . 'libraries/certs/';
+        $certsDir = '/etc/pki/tls/';
         /* See code below for logic */
         if ($ssl == CURLOPT_CAPATH) {
             $curlStatus &= curl_setopt($curlHandle, CURLOPT_CAPATH, $certsDir);
         } elseif ($ssl == CURLOPT_CAINFO) {
-            $curlStatus &= curl_setopt($curlHandle, CURLOPT_CAINFO, $certsDir . 'cacert.pem');
+            $curlStatus &= curl_setopt($curlHandle, CURLOPT_CAINFO, $certsDir . 'cert.pem');
         }
 
         $curlStatus &= curl_setopt($curlHandle, CURLOPT_RETURNTRANSFER, true);
